src/index.ts

```diff
@@ -86,6 +89,94 @@ const {grpc} = new gax.GrpcClient();
  * @property {Constructor} [promise] Custom promise module to use instead of
  *     native Promises.
  */
+export interface Client {
+  //tslint: disable-next-line: no-any
+  [k: string]: any;
```

Comment:
I mean, this pretty much make this object an `any`

Reply:
🤦‍♂️ Yeah, the precise index signature would've been even more offensive. I removed it for now to see if later PR's can help solve the issue (it's born strictly out of enabling noImplicitAny)